packages/hash/shared/src/ProsemirrorSchemaManager.ts

```diff
@@ -66,6 +66,7 @@ type ComponentNodeViewFactory = (meta: BlockMeta) => NodeViewFactory;
 export class ProsemirrorSchemaManager {
   constructor(
     public schema: Schema,
+    public accountId: string,
```

Comment:
This can be private – lots of this code isn't good at declaring whether it should be private or not because it comes from JS, but no reason not to be private 

```suggestion
    private accountId: string,
```